bundles/org.openhab.binding.shelly/README.md

```diff
@@ -0,0 +1,437 @@
+
+# Shelly Binding (org.openhab.binding.shelly)
+
+This openHAB 2 Binding implements control for the Shelly series of devices.
+This includes sending commands to the devices as well as reding the device status and sensor data.
+
+## Supported Devices
+
+|Thing               |Type                                                    | Status                                                   |
```

Comment:
I'd suggest to remove the "Status" column - we should only list supported devices (that's what the section title says).

Reply:
done